__tests__/app.test.js

```diff
@@ -29,7 +29,27 @@ describe('03_separation-of-concerns-demo routes', () => {
       });
   });
 
-  // it('gets an order by id', async () => {
+  it('gets an order by id', async () => {
+    const order = await Order.insert({ quantity: 10 });
+
+    return request(app)
+      .get(`/api/v1/orders/${order.id}`)
+      .then((res) => {
+        expect(res.body).toEqual(order);
+      });
+  });
+
+  it('gets an all orders', async () => {
```

Comment:
nice job, but noticed this test is operating with the getbyID route.  Looking for it to hit a different routes that returns all orders.